docs/current/guides/snippets/openshift-gitlab/.gitlab-ci.yml

```diff
@@ -0,0 +1,17 @@
+stages:
+  - build
+
+.dagger:
+  image: golang:1.20
+  tags:
+    - dagger
+  variables:
+    "_EXPERIMENTAL_DAGGER_RUNNER_HOST": "unix:///var/run/dagger/buildkitd.sock"  
+  before_script:
+    - cd /tmp 
+    - curl -sfL https://releases.dagger.io/dagger/install.sh | BIN_DIR=/tmp sh
```

Comment:
Note that this will install the latest release (which might not match the one pinned in values.yaml. Maybe better to pin this to the same version?

Reply:
Good point. For now I pinned the version to 0.9.1